loom/script/native/lsNativeInterface.cpp

```diff
@@ -204,7 +204,8 @@ void NativeInterface::resolveScriptType(Type *type)
         }
     }
 
-    lmAssert(0, "Unable to resolve script type %s", type->getFullName().c_str());
+    type->setMissing("resolveScriptType");
```

Comment:
If not set to ignore these, a more explanatory one gets generated down the line saying more or less the same thing including full type name. See https://github.com/LoomSDK/LoomSDK/pull/467/files#diff-540737a86ba48ee3bf2589c62397d785R167